pkg/api/networkservice/connectioncontext.proto

```diff
@@ -79,8 +79,16 @@ message EthernetContext {
     string dst_mac = 2;
 }
 
+message MetricsContext {
+    /* Enable or Disable receiving of statistics events*/
+    bool enabled = 1;
+    /* Interval to receive events in */
+    int64 interval = 2;
```

Comment:
This seems to be a source of complexity down the line.  If rather than adding a new interval here, and thus implicitly a new timer we simply 'report metrics when responding to a Request'... things become much much much simpler.

Reply:
Ok, Request based is look more simple. Thanks.